openshift/deploy/deploymentconfig/ciip-postgres-master.yml

```
@@ -78,6 +83,16 @@ objects:
                   value: 8GB
                 - name: PGDATA
                   value: /var/lib/pgsql/data/userdata
+                - name: POSTGRESQL_WAL_LEVEL
+                  value: "replica"
+                - name: POSTGRESQL_ARCHIVE_MODE
+                  value: "on"
+                - name: POSTGRESQL_ARCHIVE_TIMEOUT
+                  value: "60"
+                - name: POSTGRESQL_ARCHIVE_COMMAND
+                  value: "wal-g wal-push %p"
+                - name: WALG_GS_PREFIX
+                  value: "gs://${OC_PROJECT}-${PREFIX}postgres-walg"
```

```suggestion
                  value: "gs://${OC_PROJECT}-${PREFIX}ciip-postgres-walg"
```
the prefix is `cas-` (defined in the makefile). All of this will be much cleaner when we have helm charts in this repo